### DISCUSSION TITLE
custom javascript function

### DISCUSSION THREAD

**Discussion by akshayharshan** [Budibase/budibase#7163] [Help] [answered]
is it possible to take the value from outside the form builder using a custom javascript function?

**Comment by melohagan** [comment]
Hey @akshayharshan 

When you say custom javascript function, do you mean JavaScript bindings? To access form field values, it'll need to be from a component that is nested within the form. If the scope is an issue, then you can place the Form component at the root level (very top) of your screen component tree.

Or do you mean JavaScript within an embed component? You should still be able to access form field value bindings within this component.

**Comment by akshayharshan** [comment]
Hi @melohagan  
Thank you for your reply, let me explain my situation actually I have to display the budibase form as a feedback form on a website that is currently running. My website has multiple users and each of them has different designations I have to auto-populate each user's designation inside the feedback form field when they fill the form.
I had looked into the javascript bindings but the code will execute on the server side. 
Do you have any suggestions?

**Comment by akshayharshan** [comment, marked as answer]
got the answer, we can put pass the variables through URL like /clients/:id/:companyId/:timestamp/:customName/:etc

 **/: id/** this will act as a placeholder for the variables and those variables can be used to check conditions with the help of handlebars and change the forms styles and all.
